src/main/kotlin/week1/racingcar/RacingGame.kt

```diff
@@ -0,0 +1,36 @@
+package week1.racingcar
+
+class RacingGame {
+
+    private var numberOfCars = 0
+    private var timesToRace = 0
+
+    fun start() {
+        InputView().startInput().also {
+            this@RacingGame.apply {
+                numberOfCars = it.numberOfCars
+                timesToRace = it.timesToRace
+            }
+            race()
+        }
+    }
+
+    private fun race() {
+        val resultView = ResultView()
+        val cars = List(numberOfCars) { Car() }
+        (0 until timesToRace).forEach { _ ->
+            cars.forEach {
+                if (isPossibleToDrive()) {
+                    it.go()
+                }
+            }
+            resultView.showResult(cars)
+        }
```

Comment:
개인적으로 이 부분은 repeat 함수를 쓰는게 조금 더 가독성이 좋은 것 같아요. 🤔

```suggestion
        repeat(timesToRace) {
            cars.forEach {
                if (isPossibleToDrive()) {
                    it.go()
                }
            }
            resultView.showResult(cars)
        }
```

Reply:
이런거 없나 했는데 딱입니다 ㅎㅎ repeat을 쓰는게 더 깔끔하네요!